src/classes/base-service-call-feature.ts

```diff
@@ -114,6 +117,11 @@ export class BaseServiceCallFeature extends LitElement {
 		}
 	}
 
+	callFireDomEvent(action: IAction) {
```

Comment:
Rename to `fireDomEvent` to be consistent with other action functions.

Reply:
done